docs/usr_doc_chem_align.rst

```diff
@@ -0,0 +1,194 @@
+..
+    : The Procrustes library provides a set of functions for transforming
+    : a matrix to make it as similar as possible to a target matrix.
+    :
+    : Copyright (C) 2017-2021 The QC-Devs Community
+    :
+    : This file is part of Procrustes.
+    :
+    : Procrustes is free software; you can redistribute it and/or
+    : modify it under the terms of the GNU General Public License
+    : as published by the Free Software Foundation; either version 3
+    : of the License, or (at your option) any later version.
+    :
+    : Procrustes is distributed in the hope that it will be useful,
+    : but WITHOUT ANY WARRANTY; without even the implied warranty of
+    : MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    : GNU General Public License for more details.
+    :
+    : You should have received a copy of the GNU General Public License
+    : along with this program; if not, see <http://www.gnu.org/licenses/>
+    :
+    : --
+
+
+Chemical Structure Alignment
+============================
+
+Molecular alignment is a fundamental problem in cheminformatics and can be used for structure
+determination, similarity based searching, and ligand-based drug design et al. This problem can be
+solved by orthogonal Procrustes when given two matrices representing three-dimensional
```

Comment:
"... solved as a orthogonal Procrustes problem ..."

Reply:
Thanks for this. It's resolved in 5239ae4.